relays/client-substrate/src/metrics/float_storage_value.rs

```diff
@@ -47,13 +51,20 @@ impl<C: Chain, T: Decode + FixedPointNumber> FloatStorageValueMetric<C, T> {
 		name: String,
 		help: String,
 	) -> Result<Self, PrometheusError> {
+		let shared_value_ref = Arc::new(RwLock::new(None));
 		Ok(FloatStorageValueMetric {
 			client,
 			storage_key,
 			maybe_default_value,
 			metric: register(Gauge::new(metric_name(prefix, &name), help)?, registry)?,
+			shared_value_ref,
 		})
 	}
+
+	/// Get shared reference to metric value.
+	pub fn shared_value_ref(&self) -> F64SharedRef {
```

Comment:
I actually meant to use the `get` method from the `Gauge` metric and have that shared, instead of having a shared ref, the previous name was okay :P

Reply:
Ok - I've reverted the commit :) I do not want to expose prometheus-specific value here, because we don't have full control over that value. Like in the very beginning we've had problems with CPU metrics, because endpoint was returning `NaN` instead of some real-world value (which I can't recall now, but pretty sure it was a number). We have full control over our own shared value && it is guaranteed to be a positive number (at least for conversion rates).